src/position.cpp

```diff
@@ -684,10 +684,10 @@ void Position::do_move(Move m, StateInfo& newSt, bool givesCheck) {
     ++st->pliesFromNull;
 
     // Used by NNUE
-    st->accumulator.computed[WHITE] = false;
-    st->accumulator.computed[BLACK] = false;
-    auto& dp                        = st->dirtyPiece;
-    dp.dirty_num                    = 1;
+    st->accumulatorBig.computed[WHITE]     = st->accumulatorBig.computed[BLACK] =
```

Comment:
this should loop over the list of all network types to prevent bugs in the future

Reply:
Not sure what you mean.

Reply:
```
auto NETWORK_TYPES[] = { Big, Small };
...


for (auto color : { WHITE, BLACK })
    for (auto tp : NETWORK_TYPES)
        st->computed(color, tp) = false
```

Reply:
I c.  Feels a bit over engineered but I don't really object.  I guess we would also have to add a StateInfo::computed(Color, NetSize) method.  I was actually able to remove all the other StateInfo methods to access the accumulators as a result of your other suggestion.

Reply:
I guess it's fine, other places are also not completely generic